src/test/java/com/tozny/sdk/RealmApiTest.java

```diff
@@ -137,6 +139,23 @@ public void testQuestionChallenge() throws IOException {
     }
 
     @Test
+    public void testLinkChallenge() throws IOException {
+        LinkChallenge challenge = this.realmApi.linkChallenge("testuser@tozny.com", TOZNY_PRODUCTION_API_URL + "?method=user.link_result", 300, "verify", false, null);
```

Comment:
Change `"testuser@tozny.com"` to `this.userEmail` since it's already pulled in from the _test.properties_ file?